src/macro_util.rs

```diff
@@ -205,6 +205,26 @@ macro_rules! trailing_field_offset {
     }};
 }
 
+/// Computes alignment of `$ty`.
+///
+/// `align_of!` produces code which is valid in a `const` context.
+// TODO(#29), TODO(https://github.com/rust-lang/rust/issues/69835): Remove this
+// `cfg` when `size_of_val_raw` is stabilized.
+#[cfg(__INTERNAL_USE_ONLY_NIGHLTY_FEATURES_IN_TESTS)]
+#[doc(hidden)] // `#[macro_export]` bypasses this module's `#[doc(hidden)]`.
+#[macro_export]
+macro_rules! align_of {
+    ($t:ty) => {{
+        #[repr(C)]
```

Comment:
Can you add a safety comment here that proves that this macro computes the correct value?